index.html

```diff
@@ -940,8 +940,18 @@ <h2 class='informative'>Vary</h2>
       <section id="transmission-fixity" class="informative">
         <h2>Transmission Fixity</h2>
         <p>
-          Transmission fixity is verified by application of the <code>Digest</code> header
-          defined in [[RFC3230]] to <code>POST</code> and <code>PUT</code> requests for <a>LDP-NR</a>s.
+          Transmission fixity may be verified by including a <code>Digest</code> header (defined in
+          [[RFC3230]]) in <code>POST</code> and <code>PUT</code> requests for <a>LDP-NR</a>s.
+        </p>
+      </section>
+      <section id="persistence-fixity" class="informative">
+        <h2>Persistence Fixity</h2>
+        <p>
+          Persistence fixity may be verified by retrieving the current checksum of an <a>LDP-NR</a> and
```

Comment:
@escowles Correct, the only thing `Want-Digest` tells you is the checksum of the content of the conveyed resource any given point in time.  Casting it in terms of "can be used to infer persistence fixity" just seems more explicit, clear, and neutral/agnostic about what "persistence fixity" means, for what it's worth.